test-script.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+echo "#### Starting check for correct Meteor installation"
+INSTALLED_METEOR_VERSION=$(meteor --version)
+if [ "$INSTALLED_METEOR_VERSION" == "Meteor $METEOR_VERSION" ]; then
+  echo "Success: Meteor version confirmed";
+else
+  echo "Error: Expected Meteor version $METEOR_VERSION. Found $INSTALLED_METEOR_VERSION"
+  exit 1;
+fi
+
+echo "#### Starting check for APP_SOURCE_DIR"
+if [ -d "$APP_SOURCE_DIR" ]; then
+  echo "Success: APP_SOURCE_DIR exists";
+else
+  echo "Error: APP_SOURCE_DIR $APP_SOURCE_DIR not found";
+  exit 1;
+fi
+
+echo "#### Starting check for APP_SOURCE_DIR owner"
+APP_SOURCE_DIR_OWNER=$(ls -ld "$APP_SOURCE_DIR" | awk '{print $3}');
+if [ "$APP_SOURCE_DIR_OWNER" == "node" ]; then
+  echo "Success: APP_SOURCE_DIR is owned by correct user `node`";
```

Comment:
Good point. I tried running the echo line by itself and it does not work. It just freezes the shell (probably trying to expand, like you said).

But when run as a whole script, it runs. 

I think it's worth removing the "`" backtick around the node to prevent inconsistencies.